### PR TITLE
docs(configuration): add onListening, update after and before in devServer

### PR DESCRIPTION
**Pull request by billy-le**
onListening was added in 3.5.0
https://github.com/webpack/webpack-dev-server/pull/1760

compiler arg for after and before functions was added in 3.3.0
https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#features-4

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -762,6 +762,24 @@ module.exports = {
...
+This option allow to developers run own scripts after run webpack-dev-server
+
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  //...
+  devServer: {
+    onListening: function(server) {
+      // do fancy stuff
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]: This isn't a great example, can we change this to:
```suggestion
      const port = server.listeningApp.address().port;
      console.log('Listening on port:', port);
```

File: src/content/configuration/dev-server.md [hunk 2]
```diff
@@ -762,6 +762,24 @@ module.exports = {
 };
 ```
 
+## `devServer.onListening`
+
+`function (server)`
+
+This option allow to developers run own scripts after run webpack-dev-server
```
Comment (EugeneHlushko) [inline src/content/configuration/dev-server.md]:
```suggestion
Provides an option to execute a custom function when `webpack-dev-server` starts listening for connections on a port.
```

**Comment by billy-le** [comment]
Thank you for the feedback! I've updated the PR as requested.

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**